src/classes/HiFiAudioAPIData.ts

```diff
@@ -193,83 +446,6 @@ export class HiFiAudioAPIData {
         this.userAttenuation = userAttenuation;
         this.userRolloff = userRolloff;
     }
-
```

Comment:
No more DIff function, the diff is happening directly when it is time to send a new payload to the server